### ISSUE TITLE
"additionalMarketplaces" price for "PENDING" or “IN_PROGRESS” offers

### ISSUE THREAD

**Issue by dominikahr** [allegro/allegro-api#9579]
Hi there, 

I was wondering at what state is the price for the additional marketplaces shown in the response. Of course once it is approved it is shown there:

![image](https://github.com/user-attachments/assets/d4ba2b36-d1a9-4e6f-a823-918ad52e46c0)

But is there the price already when it is only in the "PENDING" or “IN_PROGRESS” status? Or will that be added only after is is in the “APPROVED” status?

Thanks in advance for your help!

All the best, 

Dominika

**Comment by AureliuszBrussy** [comment]
Price should also be shown in "PENDING" or “IN_PROGRESS” status.

**Comment by dominikahr** [comment]
@AureliuszBrussy ok so when the user will add the delivery method to the foreign country to his shipping price list and the publication state filed will be either PENDING or IN_PROGRESS, we will at first automatically get the price of the offer via Allegro price converter, right? 

And can we send our own price for the additional marketplaces even when it is in the publication state PENDING or IN_PROGRESS? Or we have to wait with sending our own price until it the offer on additional marketplace is APPROVED?

**Comment by AureliuszBrussy** [comment]
I need to make sure this is the case - I directed the question to the appropriate team. I'll be back with an answer.

**Comment by dominikahr** [comment]
@AureliuszBrussy any news regarding this? :)

**Comment by AureliuszBrussy** [comment]
No, I am still waiting. As soon as I have confirmation I will come back with an answer.

**Comment by AureliuszBrussy** [comment]
@dominikahr yes, I can confirm your assumptions regarding price visibility with PENDING and IN_PROGRESS statuses (it is viable), and that once the appropriate delivery method is attached to the offer, the process of making it visible on the additional market will begin, and the price will follow the rule set in the automatic price converter. 

As long as the first price change is not reprocessed, the second price change command will fail.

**Comment by dominikahr** [comment]
@AureliuszBrussy ok so just to be sure, we need to wait until the publication state on additional marketplace is APPROVED (with the price converter) and only then we can send our own price?

**Comment by AureliuszBrussy** [comment]
@dominikahr yes, that's right. But when creating a new offer, you could set your price immediately.
